sdhook.go

```diff
@@ -28,6 +28,17 @@ const (
 	DefaultName = "default"
 )
 
+// WaitGroup for the in-flight requests by sdhook. This is global so that we
+// can wait for all pending requests from all 'StackdriverHook's when
+// logrus.Exit() is called.
+var inFlight sync.WaitGroup
+
+// Has the exit handler been registered?
+var exitHandlerRegistered bool
+
+// Mutex to protect exitHandlerRegistered
+var exitHandlerRegisterMutex sync.Mutex
```

Comment:
This seems a bit overkill - Are we really that worried about two New calls in different goros?

Reply:
That said, it's down now, so I'd leave it.